pitch.md

```diff
@@ -23,38 +86,122 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 _What might happen when a program runs? Is there a computational model that
 corresponds with your domain? How does a program interact with the
 user? What kinds of semantic (i.e., non-syntax) errors might occur, and how 
 might they be communicated to the user?_
 
+The computational model is "production rules". A program written in CSound++
+takes as input a MIDI score. This can be a prewritten MIDI file, are a live
+stream of MIDI events from a MIDI controller such as a keyboard. The events
+are then parsed into lists of parameters, including information such as pitch
+and velocity (which generally corresponds to loudness). These parameters are
+then used to instantiate a CSound++ instrument, which produces an audio signal,
+the output of the program.
+
+This method of instantiating the instruments with parameters obtained from the
+MIDI score may mean that all instruments accept several implicit parameters,
+corresponding to the information contained in a MIDI note-on event.
 
 ## Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to create instruments that are based on commonly-used patterns like frequency modulation or physical modeling. It should be harder, but possible, to create completely different kinds of instruments. This may mean exposing CSound-like primitives, but encouraging users to use the higher level constructs where possible.
+
+It should be difficult or almost impossible to produce a complete song with
+this language. The language is not a DAW; that is, it is not meant to be used
+for recording, mixing or mastering. More likely, the user would need to combine
+instruments written in this language with a fully-feature DAW to produce a
+complete song.
 
 ## Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+CSound and any DAW would fall into this domain. See above for a description of
+these tools, their advantages, and their problems.
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+I plan on leveraging CSound to implement the semantics of my language. CSound
+comes with tools for generating audio from instruments, and for interfacing
+with MIDI data. This means that the only semantics I would have to implement
+would be the translation from CSound++ to CSound, which should be quite
+feasible. This will leave lots of time for language decisions.
```

Comment:
This is an awesome aspect of your project: very minimal backend/semantic work.  I like that you've done the research to pick a project that allows you to be very focused on language design.